### PR TITLE
Scroll to top after submit a new search

### LINKED ISSUE #1455: Page doesn't scroll back to top if you search for a new term
## Description
The page does not scroll back to the top after changing the search term. It seems to work fine if you load more results, but not if you just scroll down the page. Reported by @stacimc 

## Reproduction
1. Open Chrome and shrink the browser window to a narrow size.
2. Toggle into the mobile view from Chrome devtools.
3. Search for a term (‘dog’) with All content tab selected.
4. Scroll all the way to the bottom, but don’t hit ‘Load more results’.
5. Search for a term (‘cat’).
6. Notice the page doesn’t automatically scroll back up to the top of the results

## Screenshots
https://user-images.githubusercontent.com/895819/168885648-e3696b42-cbda-4827-8800-c58ee96fbc6e.mp4

## Environment
This happens both on desktop and mobile using Firefox, Chrome, and Safari.

## Resolution
- [ ] 🙋 I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by krysal**
## Fixes

Fixes #1455 by @stacimc / @panchovm 

## Description

As indicated it adds the expected scroll, and also fix a console warning in the sources page about duplicated keys.

## Testing Instructions

See the linked issue.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by sarayourfriend — COMMENTED** [review]
Something is strange about the behavior on Firefox for this PR. Whenever I execute a new search, it takes a little while to come back because the results aren't cached (maybe a slow connection thing) and the scroll does go up but still sits below the first row of results.  It sometimes does it with cached results as well :thinking: 

https://user-images.githubusercontent.com/24264157/169783923-4ea158cf-66f1-4acf-b9c6-4c79067d2a61.mp4

If I change the behavior to `auto` instead of `smooth` it fixes the problem (or at least I can't reproduce it so far no matter what I try). It doesn't seem like the smooth scrolling is really necessary in this case given submitting a search kind of feels like a page navigation anyway, where people are already accustomed to being snapped back to the top of a page.

What do you think about switching the scroll behavior to "auto"?

Also, could we add an e2e test for this?

**Comment by krysal** [comment]
Sure, I was scratching my head for how to emulate the scrolling in playwright, it doesn't seem to have a method for it but pressing the  <kbd>KeyDown</kbd> key did the trick. I'm still not sure how to get actual responses in debug mode 🤔 I tried changing the API URL in the `test/playwright/playwright.config.ts` file but didn't work.

**Comment by sarayourfriend** [comment]
Hmmm, that's odd. I will try it out tomorrow and see if I run into the same problem :thinking: It does seem like we could add a flag for the debug script to easily allow pointing to the production API for debugging purposes.

As for the scrolling, I think using pagedown like you ended up doing works fine, but for my own future reference, did you try to use the [`mouse.scroll`](https://playwright.dev/docs/api/class-mouse#mouse-wheel) method by any chance? I'm curious if you did and it happened not to work for some reason.

**Comment by krysal** [comment]
I wasn't getting consistent results with `mouse.wheel`; the test sometimes will fail, sometimes will pass unless putting a sort of sleep in between. The `mouse.wheel` method dispatches a `wheel` event, and there is a note in docs that says:

> Wheel events may cause scrolling if they are not handled, and **this method does not wait for the scrolling to finish before returning.**

So I thought it was "safer" to use the method by keyboard (and save a line also), but we can change to use it if you consider it better anyway.

File: src/components/VHeader/VSearchBar/VSearchBar.vue [hunk 1]
```diff
@@ -8,7 +8,7 @@
       v-bind="$attrs"
       class="flex-grow search-field"
       :class="{ 'border-transparent': isHomeRoute }"
-      label-text="Openverse"
+      label-text="Search for content in Openverse"
```
Comment (obulat) [inline src/components/VHeader/VSearchBar/VSearchBar.vue:11]: Great addition! It needs i18n, though.

**Comment by sarayourfriend** [comment]
> So I thought it was "safer" to use the method by keyboard (and save a line also), but we can change to use it if you consider it better anyway.

No need, your solution makes sense given the tradeoffs of the wheel method :+1: Using the keyboard probably more closely mirrors the behavior of users using assistive technologies as well, which is a positive.

**Review by sarayourfriend — APPROVED** [review]
LGTM, just needs the translations for the new term as Olga pointed out. I'll go ahead and approve though :slightly_smiling_face:

**Review by obulat — APPROVED** [review]
The scrolling works well! And thank you for adding tests 🌟

**Review by dhruvkb — APPROVED** [review]

File: src/components/VHeader/VHeader.vue [hunk 2]
```diff
@@ -173,6 +173,7 @@ export default defineComponent({
      *     update query `q` param, fetch new media.
      */
     const handleSearch = async () => {
+      window.scrollTo({ top: 0, left: 0, behavior: 'auto' })
```
[the remark this answers is not in the document]
Comment (krysal) [reply src/components/VHeader/VHeader.vue:176]: That function is only called when the 'search' button in the header is clicked, so it should be safe 😃

**Pull request merged**